packages/shipjs/src/step/setup/CI/addGithubActions.js

```diff
@@ -0,0 +1,197 @@
+import { getGitConfig } from 'shipjs-lib';
+import runStep from '../../runStep';
+import fs from 'fs';
+import path from 'path';
+import ejs from 'ejs';
+import mkdirp from 'mkdirp';
+import { print } from '../../../util';
+import { info, warning } from '../../../color';
+
+export default ({
+  baseBranch,
+  releaseBranch,
+  manualPrepare,
+  schedulePrepare,
+  cronExpr,
+  dir,
+  dryRun,
+}) =>
+  runStep(
+    {
+      title: 'Creating GitHub Actions configuration',
+    },
+    () => {
+      const gitUserName = getGitConfig('user.name') || 'Your Name';
+      const gitUserEmail = getGitConfig('user.email') || 'your@email.com';
+
+      const log = [
+        createGithubAction({
```

Comment:
This is not important, but can you rename all the filenames and function names with `Github` to `GitHub`?  🙂